### ISSUE TITLE
Test with Tow accounts in organization mode

### ISSUE THREAD

**Issue by MO-BENCHEIKH** [aws-solutions/instance-scheduler-on-aws#544]
Hello,

i had already solution deployed in my accounts ( one hub , and all others accounts spokes in my organization ), if I want test new versions of schedule solutions without  interfere with installed stack in my organization , can I just take tow account in my organization and deploy the new version with sperate namespace  it' will be not interfered with others accounts ?
only a operation by namespace is required ?

Thank you in advance.

Regards,
M.B

**Comment by CrypticCabub** [comment]
Yes, you can install a second copy of instance scheduler using a different namespace into the same account and the two should not directly interfere with one another (although for larger deployments they may run into lambda throttling since they will both be sharing the same lambda quota limits)

That said, I would recommend perfoming testing/experimentation of new features in different accounts than your main prod accounts. While Instance Scheduler is designed to be able to run 2 or more copies on the same account, best practice would be to avoid mixing experimental and production workloads.

**Comment by MO-BENCHEIKH** [comment]
tank you for reply 👍
